tests/TestCase/Database/Schema/SqliteSchemaTest.php

```diff
@@ -261,6 +261,47 @@ protected function _createTables($connection): void
         $connection->execute('CREATE INDEX "created_idx" ON "schema_articles" ("created")');
         $connection->execute('CREATE UNIQUE INDEX "unique_id_idx" ON "schema_articles" ("unique_id")');
 
+        $table = <<<SQL
+CREATE TABLE schema_no_rowid_pk (
```

Comment:
Newer PHP supports intending newdoc and will trim the matching indent from all lines.

Reply:
I could do that, but I have to do it to all the other existing ones too then, otherwise it looks weird :)

Reply:
We don't need to change this - can update them all together later.